### PR TITLE
Rasht weather

### PR DESCRIPTION
**Pull request by hedita**

### CODE REVIEW

File: assets/js/main.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    const icon = list[0].weather[0].icon;
+    iconId.innerHTML = icon;
+  
+    const time = document.getElementById("time");
+    const now = new Date();
+    const tomorrowDate = new Date(list[5].dt *1000);
+    const tomorrowDayName = document.getElementById("tomorrow-day-name");
+    const dayNameTomorrow = tomorrowDate.toLocaleString('en-us', {weekday:'long'});
+    tomorrowDayName.innerHTML = dayNameTomorrow;
+    const dayAfterTomorrowDate = new Date(list[11].dt *100);
```
Comment (blacksrc) [inline assets/js/main.js]: `100` or `1000`?

File: assets/js/main.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    const descrip = list[0].weather[0].description;
+    description.innerHTML = descrip;
+
+    const iconId = document.getElementById("icon");
+    const icon = list[0].weather[0].icon;
+    iconId.innerHTML = icon;
+  
+    const time = document.getElementById("time");
+    const now = new Date();
+    const tomorrowDate = new Date(list[5].dt *1000);
```
Comment (blacksrc) [inline assets/js/main.js:38]: `100` or `1000`?

File: assets/js/main.js [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+    
+    const dayAfterTomorrowDayName = document.getElementById("day-after-tomorrow-day-name");
+    const DayNameDayAfterTomorrow = dayAfterTomorrowDate.toLocaleString('en-us', {weekday:'long'});
+    dayAfterTomorrowDayName.innerHTML = DayNameDayAfterTomorrow;
+    
+    const year = now.getFullYear();
+    const month = now.toLocaleString('default', { month: 'short' });
+    const dayName = now.toLocaleString('en-us', {weekday:'long'});
+    const dayNumber = now.getUTCDate();
+    const date = dayName + ", " + dayNumber + " " + month + " " + year;
```
Comment (blacksrc) [inline assets/js/main.js]:
```suggestion
    const date = `${dayName}, ${dayNumber} ${month} ${year}`;
```

File: assets/js/main.js [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+    tomorrowDayName.innerHTML = dayNameTomorrow;
+    const dayAfterTomorrowDate = new Date(list[11].dt *100);
+    
+    const dayAfterTomorrowDayName = document.getElementById("day-after-tomorrow-day-name");
+    const DayNameDayAfterTomorrow = dayAfterTomorrowDate.toLocaleString('en-us', {weekday:'long'});
+    dayAfterTomorrowDayName.innerHTML = DayNameDayAfterTomorrow;
+    
+    const year = now.getFullYear();
+    const month = now.toLocaleString('default', { month: 'short' });
+    const dayName = now.toLocaleString('en-us', {weekday:'long'});
```
Comment (blacksrc) [inline assets/js/main.js]:
```suggestion
    const dayName = now.toLocaleString('en-us', { weekday: 'long' });
```

File: assets/js/main.js [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+    const dayNameTomorrow = tomorrowDate.toLocaleString('en-us', {weekday:'long'});
+    tomorrowDayName.innerHTML = dayNameTomorrow;
+    const dayAfterTomorrowDate = new Date(list[11].dt *100);
+    
+    const dayAfterTomorrowDayName = document.getElementById("day-after-tomorrow-day-name");
+    const DayNameDayAfterTomorrow = dayAfterTomorrowDate.toLocaleString('en-us', {weekday:'long'});
+    dayAfterTomorrowDayName.innerHTML = DayNameDayAfterTomorrow;
+    
+    const year = now.getFullYear();
+    const month = now.toLocaleString('default', { month: 'short' });
```
Comment (blacksrc) [inline assets/js/main.js]: check the whole js file. Either use `"` (recommended) or `'`.

File: assets/js/main.js (hunk 1 above)
Comment (hedita) [reply assets/js/main.js]: 1000.

File: assets/js/main.js (hunk 2 above)
Comment (hedita) [reply assets/js/main.js:38]: 1000.

File: assets/js/main.js [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+    const DayNameDayAfterTomorrow = dayAfterTomorrowDate.toLocaleString('en-us', { weekday: 'long' });
+    dayAfterTomorrowDayName.innerHTML = DayNameDayAfterTomorrow;
+    
+    const year = now.getFullYear();
+    const month = now.toLocaleString('default', { month: 'short' });
+    const dayName = now.toLocaleString('en-us', { weekday: 'long' });
+    const dayNumber = now.getUTCDate();
+    const date = `${dayName}, ${dayNumber} ${month} ${year}`;
+    time.innerHTML = date;  
+   console.log(list)
```
Comment (blacksrc) [inline assets/js/main.js]: remove this.

File: assets/js/main.js (hunk 5 above)
Comment (hedita) [reply assets/js/main.js]: fixed.

File: assets/js/main.js (hunk 6 above)
Comment (hedita) [reply assets/js/main.js]: done.

**Pull request merged**